docs/reference/platforms/android.rst

```diff
@@ -101,15 +101,49 @@ Additional options
 The following options can be provided at the command line when producing
 Android projects
 
-build
------
+run
+---
 
 ``-d <device>`` / ``--device <device>``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The device simulator to target. Can be either a device ID, or a device name.
+The device or emulator to target. Can be specified as:
 
-run
----
+* ``@`` followed by an AVD name (e.g., ``@beePhone``); or
+* a device ID (a hexadecimal identifier associated with a specific hardware device);
+  or
+* a JSON dictionary specifying the properties of a device that will be created.
+  This dictionary must have, at a minimum, an AVD name::
+
+     $ briefcase run -d '{"avd":"new-device"}'
+
+  You may also specify:
+
+  - ``device_type`` (e.g., ``pixel``) - the type of device to emulate
+  - ``skin`` (e.g., ``pixel_3a``) - the skin to apply to the emulator
+  - ``system_image`` (e.g., ``system-images;android-31;default;arm64-v8a``) - the Android
+    system image to use in the emulator.
+
+  If any of these attributes are *not* specified, they will fall back
+  to reasonable defaults.
+
+``-X=<value>``
+~~~~~~~~~~~~~~
+
+A configuration argument to be passed to the emulator on startup. For example,
+to start the emulator in "headless" mode (i.e., without a display window),
+specify ``-X=-no-window``. See `the Android documentation
```

Comment:
Is there some precedent for `-X`? If not, I'd prefer something more explicit like `--emu-arg`.

Reply:
The precedent I was using was clang; it has a [bunch of `-X` arguments](https://clang.llvm.org/docs/ClangCommandLineReference.html) that are all of the form "pass arg to <underlying command>" (e.g., `-Xarch_device`, `-Xcuda-ptxas`, ...), as well as the `-Wl,<arg>` commands to pass through to the linker. 

I kind of like the `X` prefix to highlight "this isn't a Briefcase command - you're talking direct to the metal"; but I can definitely get on board with including the detail of *which* metal you're talking to.